developer-docs-site/docs/guides/move-guides/move-on-aptos.md

```diff
@@ -48,24 +48,24 @@ With much more coming soon...
 
 ## Key Concepts in Move
 
-* Data should be stored within the account that owns it not the account that published the module.
-* Data flow should have minimal constraints with an emphasis on ecosystem usability
-* Prefer static type-safety over run-time safety via generics
-* A `signer` should be required to restrict access to adding or removing assets to an account unless it is explicitly clear
+* Data should be stored within the account that owns it, not the account that published the module.
+* Data flow should have minimal constraints with an emphasis on ecosystem usability.
+* Prefer static type-safety over run-time safety via generics.
+* A `signer` should be required to restrict access to adding or removing assets to an account unless it is explicitly clear.
 
 ### Data ownership
 
-Data should be stored within the account that owns it not the account that published the module.
+Data should be stored within the account that owns it, not the account that published the module.
 
 In Solidity, data is stored within the namespace of the account that created the contract. Typically this is represented by a map of an address to a value or of an instance id to the address of the owner.
 
 In Solana, data is stored within a distinct account associated with a contract.
 
-In Move, data can be stored within the module owner's account, but that creates the issue of ownership ambiguity and implies that two issues:
-1. It makes ownership ambiguous as the asset has no resource associated with the owner
-2. The module creator takes responsibility for the lifetime of that resources, e.g., rent, reclamation, etc
+In Move, data can be stored within the module owner's account, but that creates two issues:
+1. Ownership is ambiguous as the asset has no resource associated with the owner, and
+2. The module creator must take responsibility for the lifetime of that resources, e.g., rent, reclamation, etc.
 
-On the first point, by placing assets within trusted resources within an account, the owner can ensure that even a maliciously programmed module will be unable to modify those assets. In Move, we can program a standard orderbook structure and interface that would let applications built on top be unable to gain backdoor access to an account or its orderbook entries.
+On the first point, by placing assets within trusted resources within an account, the owner can ensure that even a maliciously programmed module will not be able to modify the assets. In Move, we can program a standard orderbook structure and interface that would prevent applications built on top from gaining backdoor access to an account or its orderbook entries.
```

Comment:
1. I suggest change the "the assets" to "such assets" or retain "those assets."
2. Great catch on the double negative. Thanks!

Reply:
1. Sounds good. 
2. 👍